server/cmd/dcrdex/sample-dcrdex.conf

```
@@ -0,0 +1,179 @@
+; Setting to 0 will keep all. Default is 32.
+; maxlogzips=32
+
+; ------------------------------------------------------------------------------
+; Network settings
+; ------------------------------------------------------------------------------
+
+; Use testnet.
+; testnet=true
+
+; Use simnet.
+; simnet=true
+
+; ------------------------------------------------------------------------------
+; RPC server settings
+; ------------------------------------------------------------------------------
+
+; RPC server TLS certificate file.
+; Relative to --appdata or absolute path.
+; rpccert=rpc.cert
+
+; RPC server TLS private key file.
+; Relative to --appdata or absolute path.
+; rpckey=rpc.key
+
+; IP addresses on which the RPC server should listen for incoming connections.
+; Default is 127.0.0.1:7232. 
+; rpclisten=127.0.0.1:7232
+
+; A list of hostnames to include in the RPC certificate (X509v3 Subject 
+; Alternative Name)
+; altdnsnames=
+
+; ------------------------------------------------------------------------------
+; Registration fee settings
+; ------------------------------------------------------------------------------
+
+; The extended public key for deriving Decred addresses to which DEX 
+; registration fees should be paid.
+; regfeexpub=
+
+; The number of confirmations required to consider a registration fee paid.
+; Default value is 4.
+; regfeeconfirms=4
+
+; The registration fee amount in atoms.
+; Default value is 1 DCR = 1e8 atoms.
+; regfeeamount=100000000
+
+; ------------------------------------------------------------------------------
+; PostgreSQL settings
+; ------------------------------------------------------------------------------
+
+; PostgreSQL DB name.
+; Default value is network specific.
+; pgdbname=dcrdex_{network}
+
+; PostgreSQL DB user.
+; Default value is "dcrdex"
+; pguser=dcrdex
+
+; PostgreSQL DB password.
+; pgpass=
+
+; PostgreSQL server host:port or UNIX socket (e.g. /run/postgresql).
+; Default value is 127.0.0.1:5432
+; pghost=127.0.0.1:5432
+
+; Logs the PostgreSQL db configuration on start up.
+; showpgconfig=false
+
+; ------------------------------------------------------------------------------
+; Admin server settings
+; ------------------------------------------------------------------------------
+
+; Turn on the admin server.
+; adminsrvon=false
+
+; Administration HTTPS server address.
+; Default value is 127.0.0.1:6542.
+; adminsrvaddr=127.0.0.1:6542
+
+; Admin server password. INSECURE. Do not set unless absolutely necessary.
+; adminsrvpass=
+
+; ------------------------------------------------------------------------------
+; General settings
+; ------------------------------------------------------------------------------
+
+; Display version information and exit
+; version=true
+
+; Path to the markets configuration JSON file.
+; Absolute path or relative to --appdata.
+; marketsconfpath=markets.json
+
+; Path to the markets configuration JSON file.
```

<s>Path to the markets configuration JSON file</s>

```suggestion
; The broadcast timeout specifies how long clients have to broadcast an expected
; transaction when it is their turn to act. Matches without the expected action by
; this time are revoked and the actor is penalized.
```

The text in the `--help` action is actually in accurate: "How long clients have to broadcast expected swap transactions following new blocks". So it would be good to revise that too.